.github/workflows/testBuildDeploy.yml

```diff
@@ -18,6 +18,11 @@ jobs:
         uses: actions/npm@59b64a598378f31e49cb76f27d6f3312b582f680
         with:
           args: run lint
+      - name: Run Quality Analysis on SonarCloud
```

Comment:
Maybe follow sentence case, which we're doing everything else. Only the first word and proper nous are capitalized in sentence case.

Sentence case (preferred): Run quality analysis on SonarCloud
Title case: Run Quality Analysis on SonarCloud

Reply:
Done done! Thanks!